source-code/sdk-js/src/adapter-sveltekit/ast-transformation/transforms/+layout.svelte.test.ts

```diff
@@ -1,205 +1,203 @@
-				const code = transformLayoutSvelte({} as TransformConfig, dedent`
-				<script>
-				console.log(1)
-				</script>
-				`, true)
+			// NOTES @ivan this test below is not how I understood it... Currently it will insert a slot in this case
+			// NOTES @ivan can you please look at the position of the console log in the script tag below?
+			it.skip("doesn't output markup if no markup is present", async () => {
+				const code = await transformLayoutSvelte(
+					{} as TransformConfig,
+					dedent`
+<script>
+console.log(1)
+</script>
+				`,
+					true,
+				)
 				expect(code).toMatchInlineSnapshot(`
-					"
-					<script>
-						import { getRuntimeFromData } from \\"@inlang/sdk-js/adapter-sveltekit/shared\\"
-						import { localStorageKey, getRuntimeFromContext, addRuntimeToContext } from \\"@inlang/sdk-js/adapter-sveltekit/client/reactive\\"
-						import { browser } from \\"$app/environment\\"
-
-						export let data
-
-						addRuntimeToContext(getRuntimeFromData(data))
-						let { i, language } = getRuntimeFromContext()
-
-					$: if (browser && $language) {
-						document.body.parentElement?.setAttribute(\\"lang\\", $language)
-						// TODO: only if localStorageDetector
-						localStorage.setItem(localStorageKey, $language)
-					}
-					console.log(1)
-					</script>
-					"
+"<script>import { browser } from \\"$app/environment\\";
+import { localStorageKey, getRuntimeFromContext, addRuntimeToContext } from \\"@inlang/sdk-js/adapter-sveltekit/client/reactive\\";
+import { getRuntimeFromData } from \\"@inlang/sdk-js/adapter-sveltekit/shared\\";
+console.log(1)
+export let data;
+let language;
+addRuntimeToContext(getRuntimeFromData(data));
+
+({
+  language: language
+} = getRuntimeFromContext());
+
+$:
+if (browser && $language) {
+  document.body.parentElement?.setAttribute(\\"lang\\", $language);
+  localStorage.setItem(localStorageKey, $language);
+}</script>"
 				`)
 			})
 		})
 
 		describe("root=false", () => {
-			it.skip("is a proxy for transformSvelte", async () => {
+			it("is a proxy for transformSvelte", async () => {
 				const config = {} as TransformConfig
 				const input = dedent`
-				<script>
-					export let data
-				</script>
+<script>
+    export let data
+</script>
 
-				<h1>Hello {data.name}!</h1>
+<h1>Hello {data.name}!</h1>
 				`
-				const code = transformLayoutSvelte(config, input, false)
+				const code = await transformLayoutSvelte(config, input, false)
 				expect(code).toMatch(await transformSvelte(config, input))
 			})
 		})
 	})
 
 	describe("transform @inlang/sdk-js", () => {
-		it.skip("resolves imports correctly", () => {
-			const code = transformLayoutSvelte({} as TransformConfig, dedent`
-			<script>
-				import { languages, i } from "@inlang/sdk-js"
-
-				console.log(languages)
-			</script>
-
-			{i('hello)}
-
-			`, true)
+		it("resolves imports correctly", async () => {
+			const code = await transformLayoutSvelte(
```

Comment:
please add a `;` before the `(` bracket. Just to ensure that it is valid JS if the `;` is missing in the line above